packages/collector/test/tracing/protocols/http/proxy/expressProxy.js

```diff
@@ -40,38 +39,17 @@ app.use((req, res) => {
       url = `http://localhost:${process.env.UPSTREAM_PORT}/proxy-call${req.url}`;
     }
 
-    if (req.query.httpLib === 'node-fetch') {
```

Comment:
`httpLib` can be removed from packages/collector/test/tracing/protocols/http/proxy/expressProxyControls.js#77